### PR TITLE
I3678 right curly was misconfigured for google_checks.xml

### PR DESCRIPTION
**Pull request by romani**
#3678 

InputOperatorWrapAssign.java was removed as it was used only in wrong IT method that was also removed.

### CODE REVIEW

**Comment by romani** [comment]
@MEZk , please provide code review.
I will do 7.4 release on 30th December , I want this PR to be released in 7.4.

File: src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ protected static Configuration getConfiguration() throws CheckstyleException {
      * @param clazz check class.
      * @return {@link DefaultConfiguration} instance.
      */
-    protected static DefaultConfiguration createCheckConfig(Class<?> clazz) {
+    private static DefaultConfiguration createCheckConfig(Class<?> clazz) {
```
Comment (MEZk) [inline src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java]: Can be moved closer to https://github.com/checkstyle/checkstyle/blob/dd178c43a1bc6bee90081db7aa595cb0c51974b7/src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java#L116

File: src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java [hunk 2]
```diff
@@ -254,13 +254,43 @@ protected String getCheckMessage(Map<String, String> messages, String messageKey
...
+        final Configuration result;
         final List<Configuration> configs = getCheckConfigs(checkName);
-        if (configs.isEmpty()) {
-            return null;
+        if (configs.size() == 1) {
+            result = configs.get(0);
         }
         else {
-            return configs.get(0);
+            throw new IllegalStateException("multiple instances of the same Check is detected");
```
Comment (MEZk) [inline src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java]: multiple instances of the same Check **are** detected

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java [hunk 3]
```diff
@@ -41,7 +41,8 @@ public void annotationTest() throws Exception {
 
         final Class<AnnotationLocationCheck> clazz = AnnotationLocationCheck.class;
         getCheckMessage(clazz, "annotation.location.alone");
-        final Configuration checkConfig = getCheckConfig("AnnotationLocation");
+        final Configuration checkConfig = getCheckConfig("AnnotationLocation",
+                "AnnotationLocationMostCases");
 
         final String msgLocationAlone = "annotation.location.alone";
```
Comment (MEZk) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java]: Is it a dead code?

Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java]: it is used in line 50.

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java [hunk 4]
```diff
@@ -64,4 +65,24 @@ public void annotationTest() throws Exception {
...
+
+    @Test
+    public void annotationTestVariables() throws Exception {
+
+        final Class<AnnotationLocationCheck> clazz = AnnotationLocationCheck.class;
+        getCheckMessage(clazz, "annotation.location.alone");
+        final Configuration checkConfig = getCheckConfig("AnnotationLocation",
+                "AnnotationLocationVariables");
+
+        final String msgLocationAlone = "annotation.location.alone";
```
Comment (MEZk) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java]: This is the variable which is not used in code.

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java (hunk 3 above)
Comment (MEZk) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java]: Sorry, wrong line number. See line  77

Comment (MEZk) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java]: Sorry, wrong line number. See line 77.

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java (hunk 4 above)
Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule485annotations/AnnotationLocationTest.java]: fixed

File: src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java (hunk 2 above)
Comment (romani) [inline src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java]: fixed

File: src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java (hunk 1 above)
Comment (romani) [inline src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java]: postponed till MethodDistance Check appear (base on our experiments in separate project).
I want to keep change simple in this commit to show reason why it become private.

**Comment by romani** [comment]
commits were redone and merged directly to master

**Pull request closed without merging**